src/util/dynamicConfigInjector.ts

```diff
@@ -0,0 +1,77 @@
+import cloneDeep from "lodash/cloneDeep";
+import { ProcessorRequest, RouterRequestData } from "../types/index";
+
+/* eslint-disable no-param-reassign */
+const get = require("get-value");
+const unset = require("unset-value");
+
+export class DynamicConfigInjectionService {
```

Comment:
A service under `util`?

Reply:
This is basically the service selector earlier we discussed this could be an utility hence placed it here @koladilip @krishna2020 should this be mode under service dir?

Reply:
Oh yeah for this one yes I confused it with serviceSerlector